styles.css

```diff
@@ -85,7 +87,7 @@ a {
   line-height: 24px;
 }
```

Comment:
- [ ] The header looks crashing in some of the screen sizes, I tried removing the `width` that has been set to the class `.frontpage section h2` and it worked just fine, please give it a try and kindly fix the width for the header in a way you find it properly 👍🏼 

| Your Design | Figma Design|
| -----------------  | ------------------- |
| ![image](https://user-images.githubusercontent.com/68030297/230242406-f5a4cc70-e38a-44c9-8324-92118e4258e0.png) | ![image](https://user-images.githubusercontent.com/68030297/230242483-443b339c-07c8-4c4c-b45c-694cbcf34ea9.png) |